### PR TITLE
Update performance page with VM considerations + smaller fixes

### PR DESCRIPTION
**Pull request by andrerom**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | N/A
| Versions      | 1.13+

### CODE REVIEW

**Comment by DominikaK** [comment]
@andrerom, please let me know if the proofing didn't twist the meaning in some way.

File: docs/guide/performance.md [hunk 1]
```diff
@@ -11,9 +11,8 @@ If you are in a hurry, the most important recommendations on this page are:
 
 - Use PHP 7.x, and dump optimized Composer autoload classmap
 - Use a full web (Nginx/Apache) server with vhost
-- Avoid shared file systems for code _(Docker for Mac/Win, VirtualBox/*, Vagrant, ..)_, or find ways to optimize/workaround the issues.
-- For clustering _(mainly relevant for production/staging)_; reduce latency to Redis/Memcached, use Varnish & [Solr](solr.md)
-
+- Avoid shared filesystems for code (Docker for Mac/Win, VirtualBox/*, Vagrant, etc.), or find ways to optimize or work around the issues.
```
Comment (andrerom) [inline docs/guide/performance.md]: note: unsure if readthedoc is affected, but I see everything after ´*´ is italic here

**Comment by andrerom** [comment]
Besides nitpick the proofing looks good.

File: docs/guide/performance.md (hunk 1 above)
Comment (DominikaK) [reply docs/guide/performance.md]: Local build looks ok, GitHub also renders it correctly in preview.

**Comment by andrerom** [comment]
👍

**Pull request merged**